webapp/src/css/enketo/medic.less

```diff
@@ -208,7 +208,7 @@
     max-width: 320px;
     display: block;
   }
-  .or-appearance-mrdt-verify {
```

Comment:
Can you confirm this is safe to remove the styling on `.or-appearance-mrdt-verify`? Should we be adding to it? `.or-appearance-mrdt-verify, .or-appearance-mrdt-image {}`

Reply:
But the next line isn't `.or-appearance-mrdt-verify`, it is another class.  Sorry, maybe I don't understand the structure of the markup.

Reply:
Initially we had one image input and the class `.or-appearance-mrdt-verify` then I have added an input for the field `timeTaken` and grouped them under `.or-appearance-mrdt-verify` so ended up replacing `.or-appearance-mrdt-verify` as the class associated with the image input which is why I needed to replace it in the css as well